File: airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/jobs/migrations/V0_57_2_001__AddRefreshJobType.java
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.db.instance.jobs.migrations;

import org.flywaydb.core.api.migration.BaseJavaMigration;
import org.flywaydb.core.api.migration.Context;
import org.jooq.DSLContext;
import org.jooq.impl.DSL;

public class V0_57_2_001__AddRefreshJobType extends BaseJavaMigration {

  @Override
  public void migrate(Context context) throws Exception {
    final DSLContext ctx = DSL.using(context.getConnection());
    ctx.alterType("job_config_type").addValue("refresh").execute();
  }

}

File: airbyte-db/db-lib/src/main/kotlin/io/airbyte/db/instance/jobs/migrations/V0_57_2_001__AddRefreshJobType.kt
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.db.instance.jobs.migrations

import org.flywaydb.core.api.migration.BaseJavaMigration
import org.flywaydb.core.api.migration.Context
import org.jooq.impl.DSL

@Suppress("ktlint:standard:class-naming")
class V0_57_2_001__AddRefreshJobType : BaseJavaMigration() {
  override fun migrate(context: Context) {
    val ctx = DSL.using(context.connection)
    ctx.alterType("job_config_type").addValue("refresh").execute()
  }
}
